src/components/Modal/README.md

```diff
@@ -364,17 +364,17 @@ class ModalExample extends React.Component {
               <Stack.Item fill>
                 <TextField
                   ref={this.bindNode}
-                  label=""
+                  label="Discount link"
```

Comment:
This is great, but I'm going to log a separate issue I've now noticed with how text field adds `<label>`, `aria-label`, and `aria-labelledby`, which are all duplicative.

Reply:
New issue: #1587